src/apps/backend/modules/account/rest-api/account-controller.ts

```diff
@@ -20,6 +28,34 @@ export class AccountController {
     },
   );
 
+  sendOTP = applicationController(
+    async (req: Request<GenerateOTPParams>, res: Response) => {
+
+      const account = await AccountService.sendOTP(req.body.contactNumber);
+
+      const accountJSON = serializeAccountAsJSON(account);
+
+      res.status(HttpStatusCodes.OK).send(accountJSON);
```

Comment:
This is incorrect. Sending OTP should not return JSON. It should probably send nothing. Since its a POST request, it should be 201 instead of 200